src/managers/StageInstanceManager.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const BaseManager = require('./BaseManager');
+const { TypeError, Error } = require('../errors');
+const StageInstance = require('../structures/StageInstance');
+const { PrivacyLevels } = require('../util/Constants');
+
+class StageInstanceManager extends BaseManager {
```

Comment:
This is missing a docstring.

Reply:
Added it in the latest commit 👍. Forgot to pull changes first pepehands